spec/features/support_links_spec.rb

```diff
@@ -20,7 +20,7 @@
                                  'and provide the service. For the purpose of data protection legislation, ' \
                                  'the DfE is the data controller for the personal data processed as part of ' \
                                  'Teaching Vacancies. Teaching Vacancies is a free and optional service for schools ' \
-                                 'to advertise teaching roles.')
+                                 'to list teaching roles.')
```

Comment:
I think we can avoid this fragile test failing again by replacing the assertion with:

```
expect(page).to have_content(I18n.t('who_we_are.about'))
```